front/src/pages/Organisation.jsx

```diff
@@ -89,6 +92,20 @@ export default function Organisation() {
         });
     }
 
+    const handleOnServiceCreated = (service) => {
+        organisation.services.push(service)
+        setOrganisation({...organisation})
+    }
+
+    const isUserAuthorized = () => {
```

Comment:
Je pense qu'on peut optimiser ta méthode
```suggestion
    const isUserAuthorized = () => {
        return data.roles.includes(Roles.PROVIDER) &&
           organisation.users.some(user => user.id === data.id);
};
```